### ISSUE TITLE
Funtion thumbCoverImage() doesn't return the thumb, only the route without the name of the thumb

### ISSUE THREAD

**Issue by jalediazb** [bludit/bludit#838]
Sorry for my english.
I’m learing to use your Bludit (excelent software, thanks). I’m delevoping a theme, and found a problem with images.

When I call a thumb with the function thumbCoverImage() dont show the image.

For example: 

```
<?php if ($page->coverImage()): ?>
<img class="" alt="" src="<?php echo $page->thumbcoverImage(); ?>" />
<?php endif ?>
```
Print this:
```<img class="" alt="" src="http://domain.com/bl-content/uploads/pages/7eb1a9aa5a82af690e70d8d22e9a71d2/thumbnails/">```

Using Bludit 3.2

**Comment by dignajar** [comment]
Hi, thanks for report it.
Here the fix: https://github.com/bludit/bludit/commit/3b33d20cf6f8bfe741aac7cdc4e2ba040ed88b73

is going to be included in the next version 3.4, but you can update the lines on your current installation.

Regards

**Comment by dignajar** [comment]
Fixed, please check the new version 3.4.0.
